sklearn/linear_model/coordinate_descent.py

```diff
@@ -132,16 +132,19 @@ def lasso_path(X, y, eps=1e-3, n_alphas=100, alphas=None,
 
     The Lasso optimization function varies for mono and multi-outputs.
 
-    For mono-output tasks it is::
+    For mono-output tasks it is
 
+    .. math::
         (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
 
-    For multi-output tasks it is::
+    For multi-output tasks it is
 
+    .. math::
         (1 / (2 * n_samples)) * ||Y - XW||^2_Fro + alpha * ||W||_21
```

Comment:
`_Fro` won't render correctly. n_samples should be rendered as text. The underscore, if nothing else, is a problem.